agent/src/main/java/com/teamscale/jacoco/agent/options/AgentOptions.java

```diff
@@ -203,6 +211,24 @@ public String getOriginalOptionsString() {
 		return originalOptionsString;
 	}
 
+	/**
+	 * Remove parts of the API key for security reasons from the options string.
+	 * String is used for logging purposes.
+	 */
+	public String getObfuscatedOptionsString() {
+		// Example for `getOriginalOptionsString()`:
+		// 		config-file=jacocoagent.properties,teamscale-access-token=unlYgehaYYYhbPAegNWV3WgjOzxkmNHn
+		Pattern pattern = Pattern.compile("(.*-access-token=)([^,]+)(.*)");
+		Matcher match = pattern.matcher(getOriginalOptionsString());
+		if (match.find()) {
+			String apiKey = match.group(2);
+			String obfuscatedApiKey = String.format("<obfuscated-api-key-prefix>%s", apiKey.substring(Math.max(0, apiKey.length()-4)));
```

Comment:
Isn't the obfuscated api key prefix supposed to consist of "*" characters? Is this resolved somewhere? From this MR, it looks like you are literally using the `<obfuscated-api-key-prefix>` string.

Reply:
I have adjusted this.